.gitignore

```diff
@@ -0,0 +1,8 @@
+# Ignore CMake cache files
+CMakeCache.txt
+**/CMakeFiles/
+**/*.cmake
+**/Makefile
+
+# Ignore shadow build folder
```

Comment:
@breakreturn you are right, got some of those comments from the Qt guys as well :). Thanks.